src/sections/Company/News-grid/index.js

```diff
@@ -1,19 +1,116 @@
-import React from "react";
+import React, { useState } from "react";
 import { Container, Row, Col } from "../../../reusecore/Layout";
 import Card from "../../../components/Card";
 import PageHeader from "../../../reusecore/PageHeader";
 import { NewsPageWrapper } from "./NewsGrid.style";
 import rss_feed_icon from "../../../assets/images/socialIcons/rss-sign.svg";
+import Button from "../../../reusecore/Button";
+import { useEffect } from "react";
+import SearchBox from "../../../reusecore/Search";
 
+
+let coverageFiltered= false;
+let pressReleaseFiltered=false;
+
+function colorchange(id) {
+  var background = document.getElementById(id).style.backgroundColor;
+  if (background == "rgb(30, 33, 23)") {
+    document.getElementById(id).style.background = "#b3b3b3";
+    document.getElementById(id).style.color = "black";
+  } else {
+    document.getElementById(id).style.background = "rgb(30, 33, 23)";
+    document.getElementById(id).style.color = "white";
+  }
```

Comment:
Using classNames to achieve the same would be preferable, is what I think.
Like `ele.classList.add()` and `ele.classList.remove()`

Reply:
working on this :+1: